lib/jsonapi/authorization/authorizing_operations_processor.rb

```diff
@@ -225,6 +248,57 @@ def related_models
           end
         end
       end
+
+      def authorize_model_includes(source_record)
+        if @request.include_directives
+          @request.include_directives.model_includes.each do |include_item|
+            authorize_include_item(@operation.resource_klass, source_record, include_item)
+          end
+        end
+      end
+
+      def authorize_include_item(resource_klass, source_record, include_item)
+        case include_item
+        when Hash
+          include_item.each do |rel_name, deep|
+            authorize_include_item(resource_klass, source_record, rel_name)
+            relationship = resource_klass._relationship(rel_name)
+            new_resource_klass = relationship.resource_klass
+            Array.wrap(
+              source_record.public_send(
+                relationship.relation_name(@operation.options[:context])
+              )
```

Comment:
This `public_send` straight to the record will short-circuit policy scoping on the resource level. And that's bad.